src/components/asset-list/RecyclerAssetList/index.tsx

```diff
@@ -0,0 +1,741 @@
+import { findIndex, get, isNil } from 'lodash';
```

Comment:
Can you remove the previous implementation?

Reply:
I was able to remove `findIndex` and `isNil`, but I'm hesitant to remove `get` because this is a complex operation and is used throughout the codebase.

It looks like [refactoring `get` should be easy](https://youmightnotneed.com/lodash/), unless there's already an existing implementation in the codebase that I should import instead?